app/routes/_landing.($lang).docs/components/footer/footer.tsx

```diff
@@ -1,22 +1,68 @@
-import { Flex } from "@mantine/core"
-import { Divider, IconCaretRight } from "@pokt-foundation/pocket-blocks"
-import { Link } from "@remix-run/react"
-import { LinksGroupProps } from "~/components/LinksGroup/LinksGroup"
+import { Flex, MantineTheme, useMantineTheme } from "@mantine/core";
+import {
+  Anchor,
+  Divider,
+  IconCaretLeft,
+  IconCaretRight,
+} from "@pokt-foundation/pocket-blocks";
+import { Link, useLocation } from "@remix-run/react";
+import { LinksGroupProps } from "~/components/LinksGroup/LinksGroup";
+import { getNextAndPrevNodesInTree } from "~/utils/docs";
 
 interface DocsFooterProps {
-  nextDoc: LinksGroupProps
+  items: LinksGroupProps[]
 }
 
-function DocsFooter({ nextDoc }: DocsFooterProps) {
+const commonLinkStyles = (theme: MantineTheme) => ({
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  padding: '0.625rem', 
+  width: "228px",
+  color: "white",
+  borderRadius: "0.5rem",
+  "&:hover": {
+    backgroundColor: theme.colors.navy[4],
+    textDecoration: "none",
+  },
+});
+
+function DocsFooter({ items }: DocsFooterProps) {
+  const theme = useMantineTheme()
+  const location = useLocation();
+  const lastPathPart = location.pathname.split("/").pop();
+  const [prev, next] = getNextAndPrevNodesInTree(
+    items,
+    items.find((ft) => lastPathPart === ft.slug),
+  );
+
   return (
     <Flex direction="column">
       <Divider color="gray" mb="sm" mt="xl" sx={{ borderTopWidth: "0.03rem" }} />
-      <Link style={{ display: "flex", alignSelf: "flex-end" }} to={nextDoc.link}>
-        {nextDoc.label}
-        <IconCaretRight />
-      </Link>
+      <Flex direction="row" justify="space-between">
+        {prev && (
+          <Anchor
```

Comment:
Can we use Link component from Remix with `prefetch="intent"` prop here?

Reply:
This is link from remix, but sure I can add the prefetch prop